client/src/components/PageDisplay/PageHtml.vue

```diff
@@ -32,7 +32,7 @@ export default {
             if (content) {
                 const vDom = document.createElement("div");
                 vDom.innerHTML = content;
-                const children = vDom.children;
+                const children = Object.values(vDom.children);
```

Comment:
Feels slightly clearer to do an Array.from?  Good catch, though, and +1 either way.

```suggestion
                const children = Array.from(vDom.children);
```